app/policies/mvi_policy.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+MviPolicy = Struct.new(:user, :mvi) do
+  def queryable?
+    user.icn.present? || required_attrs_present?(user)
+  end
+
+  private
+
+  def required_attrs_present?(user)
+    return if user.first_name.blank?
+    return if user.last_name.blank?
+    return if user.birth_date.blank?
+    return if user.ssn.blank?
+    return if user.gender.blank?
+
+    true
+  end
```

Comment:
Seems unorthodox to me to return `nil` or `true` - I would expect `false` | `true`.  Very minor.

Reply:
this may be my really old perl tendencies showing through, but what about something like this for the method (disclaimer: untested)
`%w[user.first_name user.last_name user.birth_date user.ssn user.gender].any?{&:blank?}` but you would need to negate it or the opposite approach `%w[user.first_name user.last_name user.birth_date user.ssn user.gender].all?{&:present?}`

Reply:
Yeah, I went with the `nil` return for brevity, b/c it also behaves the same as `false` in this conditional check.

But I am cool with adding in `return false if ...` for each one.  

I do lean towards the readability of the guard clauses here :)